### DISCUSSION TITLE
Searchterm for PnP Search Verticals

### DISCUSSION THREAD

**Discussion by gitulm** [microsoft-search/pnp-modern-search#3725] [Q&A]
Instead of the PnP Search Box, I use the Search Box from the Sharepoint homepage
![image](https://github.com/microsoft-search/pnp-modern-search/assets/56668705/3f5bb025-6e28-4c30-8e93-3f561e3dad6d)

Which search parameter can I use to fill PnP Search verticals? If I take the PnP Search Box, then the URL is
https://xxxx.sharepoint.com/SitePages/SC1.aspx?q={SearchBoxQuery}
What should the URL (at ?q={xxx}) be if I use Sharepoint Search Box?

**Comment by wobba** [comment]
Take a look at the documentation on connecting web parts using dynamic data: https://microsoft-search.github.io/pnp-modern-search/usage/search-results/connections/#input-query-text

**Comment by gitulm** [comment]
@wobba 
As far as I understand the token page (https://microsoft-search.github.io/pnp-modern-search/usage/search-results/tokens/), it should look like this in my start search page (Suche.aspx) in the verticals the search string is:
https://xxxxx.sharepoint.com/SitePages/Suche-iX.aspx?q={inputQueryText}
On the Search-iX.aspx page the following is written in the Available Connections:
![image](https://github.com/microsoft-search/pnp-modern-search/assets/56668705/1e1e76ff-5025-438f-9b10-5cbc2daf4f21)
If I set this up, the search string from Suche.aspx will not be passed on to the Suche-iX.aspx page through the verticals.
Am I misunderstanding something or is the token {inputQueryText} wrong?

**Reply by wobba** [reply]
This is better.
<img width="333" alt="image" src="https://github.com/microsoft-search/pnp-modern-search/assets/51104/e4ee4763-076d-4eee-bea1-7e390d61cb5b">

**Comment by gitulm** [comment]
I found the MS Search search string:
https://xxxx.sharepoint.com/SitePages/Suche.aspx?q={**QueryString.q**}

**Comment by gitulm** [comment]
QueryString.q is the right search string.
